editor/src/clj/editor/code/view.clj

```diff
@@ -2570,7 +2573,8 @@
      (schedule-hover-refresh! view-node evaluation-context)))
   ([view-node evaluation-context]
    (some-> (g/node-value view-node :hover-request evaluation-context) ui/cancel)
-   {:hover-request (ui/->future 0.2 #(refresh-hover-state! view-node))}))
+   (let [delay (if (hover-visible? view-node evaluation-context) 0.2 1.0)]
```

Comment:
This behavior is closer to vscode: initially, there is a longer delay before showing the popup. Once the popup is shown, its state is refreshed more often so it can, e.g., close faster.